at-chuongphu/excercise6.rb

```diff
@@ -0,0 +1,32 @@
+# ex1
+cong = proc { |*ar| ar.inject { |a, b| a + b } }
+tru = proc { |*ar| ar.inject { |a, b| a - b } }
+nhan = proc { |*ar| ar.inject { |a, b| a * b } }
+chia = proc do |*ar|
+  ar.inject { |a, b| a / b unless b.zero? }
+end
+p cong.(3, 1, 4)
+p tru.(7, 2)
+p nhan.(5, 2)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/4be4c2efcb7335770c89fa1e11f620eff8daeaca/at-chuongphu/excercise6.rb#L10"></span>Style/LambdaCall: Prefer the use of <code>lambda.call(...)</code> over <code>lambda.(...)</code>. (https://github.com/rubocop-hq/ruby-style-guide#proc-call)</td>
    </tr>
  </tbody>
</table>